### ISSUE TITLE
How about BN？

### ISSUE THREAD

**Issue by lliuz** [weiliu89/caffe#58]
Hello @weiliu89 

I found that there is no batchnorm used .

```
# Currently only the non batch norm version has been tested.
use_batchnorm = False
```

BN is common to accelerate training now, What is consideration for not using BN? Where should I change in codes to use BN?

Thanks.

**Comment by weiliu89** [comment]
Since I used VGG for most of my experiments and it is not trained with BN, thus I didn't optimize for BN parameters. I have tried ResNet101 (which uses BN), which has similar performance as VGG16 on VOC. By far, VGG works the best on VOC. Probably ResNet can show advantage on COCO (which has more classes and thus needs higher capacity network), but I haven't tested it yet. I will update the code to include a demo script on how to train with ResNet when I have time.
